config/locales/en.yml

```diff
@@ -3,7 +3,7 @@
 en:
   buttons:
     abandon: "Abandon this %{thing}"
-    adopt: "Adopt!"
+    adopt: "%{capitalize(c(brand.imperative))}!"
```

Comment:
The locales files don't support the c function unfortunately. These locales files are used in templates by the `t` function. For example, [here's where this string is used](https://github.com/alexjball/adopt-a-drain/blob/ec47743366b896873547e3fae2e75c72d478b2be/app/views/things/adopt.html.haml#L10)

So for locale strings we need to change, the workflow is:

1. Find where the string is referenced by searching the codebase for "buttons.adopt" or whatever the key is
2. change `t("buttons.adopt")` to `c("brand.imperative").capitalize`
3. change the locale value to something so you know you've migrated it

Reply:
To clarify, 

- you are able to use the c function in any of the .haml or .erb template files under `views`.
- You need to quote the argument you pass to the c function
- The syntax for erb files is `<%= c("site.main_url") %>` ([example](https://github.com/alexjball/adopt-a-drain/blob/ec47743366b896873547e3fae2e75c72d478b2be/app/views/adoptions_mailer/usage_report.html.erb#L2))
- The syntax for haml files is `#{c("city.title")}` ([example](https://github.com/alexjball/adopt-a-drain/blob/ec47743366b896873547e3fae2e75c72d478b2be/app/views/sidebar/_tos.html.haml#L4))

Reply:
I made some progress.  I figured out that the values in the templates will come from the brand .yml files via adopt.html.haml, instead of from the en.yml file.   
- Not sure of the format needed in the adopt.html.haml file
- Some of the values have variables embedded, so can't be taken directly from the brand files.  For example:

`update_report: # "%{title} import (%{deleted_adopted_count} %{c(past_tense_verb} %{things} removed, %{created_count} %{things} added, %{deleted_unadopted_count} un%{past_tense_verb} %{things} removed)"`

Reply:
Getting closer.  I pushed up some updates to the haml and yml files.  Need some clarification on adopt.html.haml